components/card/demo/loading.md

```diff
@@ -1,15 +1,23 @@
 ---
 order: 5
-title: 预加载的卡片
+title:
+  zh-CN: 预加载的卡片
+  en-US: Preloaded card
```

Comment:
Preloaded means that the content is pre-loaded... but below, you say it is being fetched ... which makes it sound like an asynchronous load?